ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -284,6 +284,36 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                                         requireReversal:(BOOL)requireReversal
                                                 options:(ORKPredefinedTaskOption)options;
 
+/**
+ Returns a predefined task that tests the participant's reaction time.
+ 
+ In a device motion reaction task the participant is asked to move the device sharply in any direction in response to a visual cue. You can use this task to accurately assess the participant's 'Simple Reaction Time'.
+ 
+ A device motion reaction time task finishes when the uses has completed the required 'numberOfAttempts' successfully. An attempt is successful when the user exerts a g-force greater than 'terminationGs' to the device after the stimulus has been delivered and before the timeout has elapsed. An attempt is unsuccessful if a g-force greater than 'terminationGs' is applied to the device before the stimulus or if this does not occur before the timeout interval has elapsed. If unsuccessful, the result is not reported and the user must try again to proceed with the task.
+ 
+ Data collected by the task is in the form of ORKDeviceMotionReactionTimeResult objects. These objects contain a timeInterval for the delivery of the stimulus and an ORKFileResult which references the motion data collected during an attempt. The developer / researcher may use this data to accurately determine the first movement in response to the stimulus.
+ 
+ TODO: A standard implementation of response time for convenience should the researcher / developer not wish to calculate this themselves.
```

Comment:
Explain what you had in mind for this? Doesn't belong in the header anyway - open an issue for it and link it back to this PR.